api/db/mongoose.js

```diff
@@ -1,17 +1,16 @@
 require('dotenv').config();
 const mongoose = require('mongoose');
-const seedUsers = require('../scripts/seed')
 
 const uri = process.env.DB_URI
 mongoose.connect(uri, {useNewUrlParser: true, useUnifiedTopology: true});
-seedUsers();
```

Comment:
FIX: we don't want to seed users on every connect, each time we restart the server we make DB calls not needed. This is why we use the sync.js script.